migrations/versions/0c6db53f2db2_.py

```diff
@@ -0,0 +1,42 @@
+"""empty message
+
+Revision ID: 0c6db53f2db2
+Revises: 62a65242ccf3
+Create Date: 2020-06-26 17:55:26.309606
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '0c6db53f2db2'
+down_revision = '62a65242ccf3'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('coreData', sa.Column('negativeTestsViral', sa.Integer(), nullable=True))
+    op.add_column('coreData', sa.Column('positiveCasesViral', sa.Integer(), nullable=True))
+    op.add_column('coreData', sa.Column('positiveTestsViral', sa.Integer(), nullable=True))
+    op.add_column('coreData', sa.Column('totalTestsViral', sa.Integer(), nullable=True))
+    op.drop_column('coreData', 'pcrNegativeTests')
```

Comment:
This will blank out the existing values in these columns since they're being deleted and recreated instead of renamed with something like 
`op.alter_column('coreData', 'pcrNegativeTests', nullable=False, new_column_name='negativeTestsViral')`

I'm fine with doing that if you want to do a new backfill after this deploys or we can rename, whichever you think makes the most sense.

Reply:
ack sorry you're right! Fixing this now - have to remind myself to not blindly rely on alembic...